components/kyma-operator/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -38,7 +38,7 @@ func NewClient(host string, TLSKey string, TLSCert string, TLSInsecureSkipVerify
 	}
 	tlscfg, err := tlsutil.ClientConfig(tlsopts)
 	return &Client{
-		helm: helm.NewClient(helm.Host(host), helm.WithTLS(tlscfg)),
+		helm: helm.NewClient(helm.Host(host), helm.WithTLS(tlscfg), helm.ConnectTimeout(30)),
```

Comment:
I'd love to have this configurable :D  And did you create maybe bump image pr ? ;-)

Reply:
I created a follow-up task for making it a configurable variable -  #5994. For me also other timeouts shouldn't be hardcoded. As I said in slack, this is just a hotfix.

It was bumped in this PR (all tests passed), but during the rebase of this PR, I removed a bump, because it pointed to PR image.